docs/en/notes/faq.md

```diff
@@ -214,6 +214,12 @@ Compatible MMDetection, MMEngine, and MMCV versions are shown as below. Please c
   1. According to the definition of COCO dataset, the small and medium areas in an image are less than 1024 (32\*32), 9216 (96\*96), respectively.
   2. If the corresponding area has no object, the result of AP and AR will set to -1.
 
+### How do you run per class validation?
```

Comment:
The main branch has now switched to 3.x and the usage is not this. Could you please update and give dev-3.x a pr

Reply:
Really? I'm using the 3.x branch and this is working perfectly.

Reply:
Or do you mean? 

```
val_evaluator = (...) 
```

e.g. the difference in the name of the config variable?

Reply:
val_evaluator